LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/Processors/LnsProtocolMessageProcessor.cs

```diff
@@ -190,6 +195,7 @@ internal async Task InternalHandleDataAsync(RouteValueDictionary routeValues, We
                     break;
                 case LnsMessageType.JoinRequest:
                     LogReceivedMessage(this.logger, "jreq", json, null);
+                    JoinRequestCounter.Add(1, new[] { KeyValuePair.Create(MetricRegistry.GatewayIdTagName, (object)stationEui) });
```

Comment:
We might want to track this metric later, after deduplication

Reply:
Addressed this in 98676615807f87182133e578e24f20c3ac20cd46